solarized-definitions.el

```diff
@@ -237,30 +237,45 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
                     (diff-changed ((t (,@fmt-revr ,@fg-yellow))))
                     (diff-removed ((t (,@fmt-revr ,@fg-red))))
                     (diff-refine-change
-                     ((t (,@fmt-revr ,@fg-blue ,@bg-back))))))
+                     ((t (,@fmt-revr :foreground "black" ,@bg-base01))))))
                  (low
                   `((diff-added ((t (,@fmt-undr ,@fg-green))))
                     (diff-changed ((t (,@fmt-undr ,@fg-yellow))))
                     (diff-removed ((t (,@fmt-bold ,@fg-red))))
                     (diff-refine-change
-                     ((t (,@fmt-undr ,@fg-blue ,@bg-back))))))
+                     ((t (,@fmt-undr :foreground "black" ,@bg-back))))))
                  (normal
                   (if window-system
                       `((diff-added ((t (,@fmt-bold ,@fg-green))))
                         (diff-changed ((t (,@fmt-bold ,@fg-yellow))))
                         (diff-removed ((t (,@fmt-bold ,@fg-red))))
                         (diff-refine-change
-                         ((t (,@fmt-bold ,@fg-blue ,@bg-back)))))
+                         ((t (,@fmt-bold :foreground "black" ,@bg-base01)))))
                     `((diff-added ((t (,@fg-green))))
                       (diff-changed ((t (,@fg-yellow))))
                       (diff-removed ((t (,@fg-red))))
-                      (diff-refine-change ((t (,@fg-blue ,@bg-back))))))))
+                      (diff-refine-change ((t (:foreground "black" ,@bg-base01))))))))
```

Comment:
`"black"` is not a color used in this theme.

I’m planning to support something like #191 soon, which will make it much easier to locally extend Solarized without having to modify the repo.